packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/gux-form-field-radio.tsx

```diff
@@ -66,7 +72,7 @@ export class GuxFormFieldRadio {
     return (
       <Host
         class={{
-          'gux-input-error': this.hasError,
+          'gux-input-error': this.hasError || this.hasGroupError,
           'gux-disabled': this.disabled
```

Comment:
I know you are working on the `disabled` functionality atm will  the `gux-disabled` be  `this.disabled || this.hasGroupDisabled` ?

Reply:
I think the disabled functionality should now work as expected